### PR TITLE
[RFC] Update Contributing Guide

### PR DESCRIPTION
**Pull request by CodingKoopa**
This PR updates the contributing guide to be more accurate, as it has became outdated with recent changes such as the new testcase submission system.

Changes made include:
- [x] Clean up leftovers from testcase submission.
- [x] Address title IDs of other regions.
- [x] Add note for screenshot names.
- [x] Add note for screenshot variety.
- [x] Update note for screenshot black background.
- [x] Add POV note for wiki entries.

I'm looking for feedback on any other minor things that need to be changed here.

### CODE REVIEW

**Comment by Leo626** [comment]
> Title IDs can be found near the top of a log when running a game. For example, this is what it looks like for The Legend of Zelda: Ocarina of Time, 0004000000033500: `[   0.019882] Loader <Info> core/loader/ncch.cpp:Load:340: Program ID: 0004000000033500`.

Since the compatibility ratings are based on user submitted testcases which can have different Title IDs for the same game, can we suggest to google search for "3DS Title IDs" for other regions?
___
There's also this from Discord.
discord_2018-04-28_10-55-07

**Comment by CodingKoopa** [comment]
In regards to the screenshot name change suggestion by @AuraOfTheDawn, I agree that that should be changed, but a followup PR fixing existing ones will be needed.

**Comment by AuraOfTheDawn** [comment]
Discussion was had, again, about scripting an auto-create for some of this, which would obviate the need for finding TID's and boxart, though I dunno if that's gonna go through. Been discussed.... several times before.

Slight Nit. If a game has a graphical issue (missing graphics, bad shadows, bad shaders, etc) should we suggest they include a screenshot of it? This lets people looking at the compatdb know how bad it'll get, where there's a description 'due to unsupported geometry shaders some graphics don't render correctly' rather then just hoping it's still playable to them. There's often a screenshot taken anyway when some user tells us 'hey, what's wrong here'. Add to the 'screenshot variety' note if we do this?

**Comment by chris062689** [comment]
Auto generating the game.dat file won't be too difficult, just have to find time to do it. 

The only issue I can see with showing graphical issues in screenshots is that they won't be removed when the bug is fixed, so it may be misleading to go to a games page and see outdated bug screenshots.

**Comment by CodingKoopa** [comment]
I feel that screenshots should be taken of whatever is a signifigant part of the game, unbiased by whether there are graphical glitches present or not.

**Comment by Leo626** [comment]
I agree with @CodingKoopa. Screenshots are suppose to show how the game emulates. If there are graphical issues, then they must be taken in the screenshots. The issue is remembering to update them when they are fixed.

**Comment by AuraOfTheDawn** [comment]
Yes, that was my opinion @Leo626 , but I was leaning towards a bias. If you submit data for a game and say 'it has a problem at X place', you should show that problem. Koopa was speaking against that, I think? Having to remove if the bug is fixed is a flaw, though showing dates on the photos could be a stopgap if they slip past. Or a version number on the screenshot it was taken from. That woudl be an interface change on the site though.

**Comment by CodingKoopa** [comment]
What I've seen some sites implement is a warning about old test results, for example, [Wine's App Compatibility Database](https://appdb.winehq.org/objectManager.php?sClass=version&iId=32371). A banner about outdated test results akin to the red banner warning about incomplete pages would be overkill, considering that the raw compatibility entries *will* be updated. Displaying an outdated notice in the screenshots section would also be overkill, because practically every game would get flagged for that, and there may or may not be graphical glitches present. Perhaps there could be a system for tagging which screenshots contain graphical glitches, and only monitoring those for obselence based on date?

**Comment by chris062689** [comment]
@CodingKoopa Can this be merged? Is this up to date? @j-selby

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -134,17 +135,19 @@ The required process for getting virtual console boxart is:
 The screenshots for the game are located in `/<Game Name>/screenshots/` (See: [Screenshots](#screenshots)). Screenshots **must** follow these specifications:
   - Native resolution.
   - Smallest window size.
-  - Black background (For the blank space left and right of the bottom screen.). To achieve this, go to the [User Directory](https://citra-emu.org/wiki/user-directory/), and from there navigate to the `config` directory. Open qt-config.ini with a text editor, and set bg_blue, bg_green, and bg_red to 0.
+  - Have variety. In addition to, say, the title screen, you should also have screenshots of different areas of gameplay, too, unless the game crashes before then.
```
Comment (Schplee) [inline CONTRIBUTING.md]: `you should also have screenshots of different areas of gameplay, too`
Would be less redundant if `, too` were removed.

**Comment by Leo626** [comment]
Worth adding a small note in the Wiki section to keep it objective and no use of "I"?

**Comment by CodingKoopa** [comment]
Addressed the reviews.

**Pull request merged**